### DISCUSSION TITLE
How to add custom javascript

### DISCUSSION THREAD

**Discussion by Synomega** [colinwilson/lotusdocs#112] [Feature Requests]
Hi,

I use the theme as as a Hugo Module, could you tell me the best option to add a custum js script please ?

Thanks in advance.

**Comment by colinwilson** [comment]
There isn't an efficient way to do this at the moment. But I'll add an option that automatically loads a theme users named `.js` file if it exists.

Note: I've changed the category of this post to a feature request.

**Reply by Nadine2016** [reply]
I am also interested in that feature. thank you.

**Comment by life1347** [comment]
I spent some time this weekend to add this features. Please check the PR https://github.com/colinwilson/lotusdocs/pull/150
